superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -101,6 +101,7 @@ export default function ErrorAlert({
   const [isBodyExpanded, setIsBodyExpanded] = useState(false);
 
   const isExpandable = ['explore', 'sqllab'].includes(source);
+  const isLongMessage = (subtitle as string).split(/\r\n|\r|\n/).length > 30;
```

Comment:
How compatible is this with different screen sizes/zoom levels? Just wondering if that 30 is always a solid choice.

Reply:
I wounder if we can do it based on overall length somehow?

Reply:
I think overall length/character based might be a better approach, maybe something between 128-256 characters would be the endpoint?

Reply:
Is there not some React/Typescript/Javascript magic to just make a scrollbar if stuff overflows?

Reply:
After looking again I do think it makes sense to try a better way.  But im not sure if there is a clever way to do it because we are trying to figure this out BEFORE its rendered

Reply:
Did we decide to scrap this?